test/services/billing/supplementary/fetch-previous-transactions.service.test.js

```diff
@@ -12,15 +12,22 @@ const DatabaseHelper = require('../../../support/helpers/database.helper.js')
 const BillRunHelper = require('../../../support/helpers/water/bill-run.helper.js')
```

Comment:
Feel free to ignore since not a change you've made but the order for these isn't alphabetical.